Leibit.Core/Data/nhth/NLO.xml

```diff
@@ -0,0 +1,152 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<estw>
+  <station name="Wiesthal" short="NWI" refNr="75">
+    <track calculateDelay="false" isPlatform="false">
+      <block name="76B31" direction="right"/>
+      <block name="75S34" direction="right"/>
+    </track>
+  </station>
+
+  <station name="Partenstein" short="NPAR" refNr="76" scheduleFile="g76_____.abf" localOrderFile="bf76____.abf">
+    <track name="001">
+      <block name="76G001"/>
+    </track>
+    <track name="002">
+      <block name="76G002"/>
+    </track>
+    <track name="003">
+      <block name="76G003"/>
+    </track>
+  </station>
+  
+  <station name="Lohr Stadt" short="NLOS" refNr="77">
+    <track calculateDelay="false" isPlatform="false">
+      <block name="77B01" direction="right"/>
+    </track>
+  </station>
+
+  <station name="Lohr" short="NLO" refNr="77" scheduleFile="g77_____.abf" localOrderFile="bf77____.abf">
+    <track name="001">
+      <block name="77G001"/>
+    </track>
+    <track name="002">
+      <block name="77G002"/>
+    </track>
+    <track name="003">
+      <block name="77G003"/>
+    </track>
+    <track name="004">
+      <block name="77G004"/>
+    </track>
+    <track name="005">
+      <block name="77G005"/>
+    </track>
+  </station>
+
+  <station name="Nantenbach" short="NNAN" refNr="78" scheduleFile="g78_____.abf" localOrderFile="bf78____.abf">
+    <track name="063" isPlatform="false">
+      <block name="78B63" direction="right"/>
+      <block name="78B311" direction="left"/>
+    </track>
+    <track name="108" isPlatform="false">
+      <block name="77B12" direction="left"/>
+      <block name="78B264" direction="right"/>
+      <block name="78B308" direction="right"/>
+    </track>
+    <track name="207" isPlatform="false">
+      <block name="78B207" direction="right"/>
+    </track>
+  </station>
+
+  <station name="Langenprozelten" short="NLP" refNr="79">
+    <track calculateDelay="false" isPlatform="false">
+      <block name="79B61" direction="left"/>
+      <block name="78B62" direction="left"/>
+    </track>
+  </station>
+
+  <station name="Üst Dittenbrunn" short="NDI" refNr="7">
```

Comment:
Nach der Betriebsstellenliste wird die Üst nur "Dittenbrunn" genannt.

Reply:
Danke für den Hinweis. Ich werde beizeiten die Namen überarbeiten und die überflüssigen Zusätze Üst, Abzw und Hp entfernen. Das möchte ich jetzt nicht in einer Hau-Ruck-Aktion machen, weil ich ja bis Gotha durchgehen müsste.